core-bundle/composer.json

```
@@ -67,7 +67,7 @@
         "doctrine/collections": "^2.1",
         "doctrine/dbal": "^3.6.2",
         "doctrine/doctrine-bundle": "^2.8",
-        "doctrine/orm": "^2.17",
+        "doctrine/orm": "^2.20 || ^3.4",
```

```suggestion
        "doctrine/orm": "^2.17 || ^3.4",
```
I don't see a reason in bumping this version for no reason as if has been working fine.